content/applications/finance/expenses.rst

```diff
@@ -390,8 +485,9 @@ Post expenses in accounting
 ===========================
 
 Once an expense report is approved, the next step is to post the report to the accounting journal.
-To view all expense reports to post, go to :menuselection:`Expenses --> Expense Reports --> Reports
-To Post`.
+To view all expense reports, go to :menuselection:`Expenses --> Expense Reports`. To view only the
+expense reports that have been approved and need to be posted, adjust the filters on the left
+side so that only the :guilabel:`Approved` status is active (checked off).
```

Comment:
Early line break
```suggestion
expense reports that have been approved and need to be posted, adjust the filters on the left side
so that only the :guilabel:`Approved` status is active (checked off).
```